chapter/chapter.md

```diff
@@ -16,32 +16,20 @@ actually makes its changes easier to manage.
 
 ## The Problem
 
-Users can interact with programs in a bewildering variety of ways,
-but the most popular on the web remains the HTML form.
-In a classic 1990s-style application,
-a user is presented with a form,
-fills in the fields,
-and clicks "Submit".
-This sends an HTTP POST request to a server
-with the information the user entered as its payload.
-The server validates that information,
-updates a database,
-and returns an HTML page that may or may not contain another form.
-
-As browsers have become more powerful,
-and as interaction has moved away from the desktop to mobile devices,
-developers have moved more and more of this processing cycle to the client.
-The first step was to write a little JavaScript
-to sanity-check the data the user entered before POSTing it
-so that users wouldn't waste time waiting for the server to respond,
-only to have to fix simple typos in the form.
-
-Programmers quickly began using JavaScript to make form interactions more fluid
-by showing and hiding sections based on what users had entered so far,
-and by fetching bits of data to do things like auto-completion while users were still typing.
-All of this made for a better user experience,
-but at the price of increasing client-side complexity.
-A modern single-page application (SPA) may have to keep track of hundreds of pieces of information,
+You have probably built web applications that use forms.
+They are easy enough to manage if you only have a few,
+or a few dozen,
+but manual approaches fall apart by the time you have
+hundreds of distinct forms,
+each of which can update the application's state in different ways.
```

Comment:
Perhaps it would be too much, but what would you think about mentioned the each updates the application's state and triggers multiple types of validations.
Dunno, just brainstorming that the validations is a big factor in all this and selling it in the intro might be good? 😄